core-services/image-mirroring/openshift/mapping_origin_4_3

```diff
@@ -214,3 +215,4 @@ registry.svc.ci.openshift.org/origin/4.3:thanos quay.io/openshift/origin-thanos:
 registry.svc.ci.openshift.org/origin/4.3:upi-installer quay.io/openshift/origin-upi-installer:4.3 quay.io/openshift/origin-upi-installer:4.3.0
 registry.svc.ci.openshift.org/origin/4.3:vertical-pod-autoscaler quay.io/openshift/origin-vertical-pod-autoscaler:4.3 quay.io/openshift/origin-vertical-pod-autoscaler:4.3.0
 registry.svc.ci.openshift.org/origin/4.3:vertical-pod-autoscaler-operator quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.3 quay.io/openshift/origin-vertical-pod-autoscaler-operator:4.3.0
+registry.svc.ci.openshift.org/origin/4.3:whereabouts-cni quay.io/openshift/origin-whereabouts-cni:4.3 quay.io/openshift/origin-whereabouts-cni:4.3.0
```

Comment:
This name is not consistent with our pattern.  What does this image do?

Reply:
Provides a CNI plugin -- whereabouts-cni provides dynamic IP address assignment for additional network attachments (e.g. as made by Multus CNI) across nodes in the cluster without requiring that those networks have DHCP daemons available to serve DHCP requests

Reply:
Same as the other comments, this needs to be scoped under the component or rough area (we generally don't allow "branding" names that don't clearly describe what they do).  So for example `multus-dynamic-ip-assignment-cni` or `network-dynamic-address-assignment-cni` would probably be acceptable.  We might allow `multus-whereabouts-cni` but we try to avoid names that don't describe what the thing does.

Reply:
Makes sense, I like the `multus-*` prefix in general as it provices context, in this case, I think I like the name `multus-whereabouts-ipam-cni` as it encompasses the upstream project name as well as the context under which it's used -- with additional networks as created by Multus CNI, and provides IP Address Management (IPAM)